rm_vk/rm_vk/Common/LikeControl/LikeControl.swift

```diff
@@ -0,0 +1,62 @@
+// LikeControl.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Кастомный контрол для лайка
+final class LikeControl: UIControl {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let heartImageName = "suit.heart"
+        static let fillHeartImageName = "suit.heart.fill"
+        static let secondaryLabelColorName = "secondaryLabel"
+    }
+
+    // MARK: - IBOutlet
+
+    @IBOutlet private var likeImageView: UIImageView!
+    @IBOutlet private var likeCountLabel: UILabel!
+
+    // MARK: - Private property
+
+    private var isLike = false
+    private var likeCount = 0
+
+    // MARK: - Initializers
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createGestureRecognizer()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        createGestureRecognizer()
+    }
+
+    // MARK: - Private methods
+
+    private func createGestureRecognizer() {
+        let tap = UITapGestureRecognizer(target: self, action: #selector(tapAction(_:)))
```

Comment:
```suggestion
        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapAction(_:)))
```
возможно заругают за название, но это уже от настроения зависит